compiler/rustc_hir_typeck/src/expr_use_visitor.rs

```diff
@@ -122,18 +122,12 @@ impl<'a, 'tcx> ExprUseVisitor<'a, 'tcx> {
     /// - `delegate` -- who receives the callbacks
     /// - `param_env` --- parameter environment for trait lookups (esp. pertaining to `Copy`)
     /// - `typeck_results` --- typeck results for the code being analyzed
-    pub fn new(
+    pub(crate) fn new(
         delegate: &'a mut (dyn Delegate<'tcx> + 'a),
```

Comment:
style nit: pls flip arguments: `fcx, body_id, delegate` instead of teh current order